### PR TITLE
Add :user-valid pseudo-class to the pseudo-classes list

### PR DESCRIPTION
**Pull request by fpetrakov**
### Description

Add `:user-valid` pseudo-class to the pseudo-classes list

### CODE REVIEW

File: files/en-us/web/css/pseudo-classes/index.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ These pseudo-classes relate to form elements, and enable selecting elements base
   - : Matches when a form element is required.
 - {{CSSxRef(":optional")}}
   - : Matches when a form element is optional.
+- {{CSSxRef(":user-valid")}}
+  - : Represents an element with corrent input, but only when the user has interacted with it.
```
Comment (teoli2003) [inline files/en-us/web/css/pseudo-classes/index.md:76]:
```suggestion
  - : Represents an element with correctt input, but only when the user has interacted with it.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thanks a lot!

This PR looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! ✌️

**Comment by fpetrakov** [comment]
@teoli2003 thanks!

**Pull request merged**